src/simularium/rendering/InstancedFiberShader.ts

```diff
@@ -371,39 +364,21 @@ void main()	{
     float fragPosDepth = (((f - n) * fragPosNDC.z) + n + f) / 2.0;
 
     // type id, instance id, zEye, zFragDepth
-    fragColor = vec4(IN_instanceAndTypeId.y, IN_instanceAndTypeId.x, fragViewPos.z, fragPosDepth);
-}
-`;
+    gAgentInfo = vec4(IN_instanceAndTypeId.y, IN_instanceAndTypeId.x, fragViewPos.z, fragPosDepth);
 
-const normalShader = `
-precision highp float;
-
-in vec3 IN_viewNormal;
-out vec4 fragColor;
-void main()	{
     vec3 normal = IN_viewNormal;
     normal = normalize(normal);
     vec3 normalOut = normal * 0.5 + 0.5;
-    fragColor = vec4(normalOut, 1.0);
-}
-`;
+    gNormal = vec4(normalOut, 1.0);
 
-const positionShader = `
-precision highp float;
-
-in vec3 IN_viewPos;
-out vec4 fragColor;
-void main()	{
-    
-    vec3 fragViewPos = IN_viewPos;
-    fragColor = vec4(fragViewPos.x, fragViewPos.y, fragViewPos.z, 1.0);
```

Comment:
where there were 3 fragment shaders before, now there is a single one that writes 3 different outputs